client/web/dev-sandbox/src/static/lib/index.ts

```diff
@@ -1,7 +1,20 @@
+import { WikijumpAPI } from "@wikijump/api"
 import "@wikijump/ftml-components"
 import App from "./App.svelte"
 
 window.addEventListener("DOMContentLoaded", async () => {
   const container = document.querySelector("#app")!
   const app = new App({ target: container })
+
+  setTimeout(async () => {
+    try {
+      const api = new WikijumpAPI()
+      const response = await api.userGet("id", 1234)
+      console.log("mocked API response:")
+      console.log(response)
+    } catch (err) {
+      console.warn(err)
+      console.log("API probably isn't being mocked")
+    }
```

Comment:
I don't think I understand what this snippet does.

Reply:
All it does is make an API client instance and try to make a request. If this simple request fails, it's probably because the mock server wasn't started. You have to start both Vite and Mockoon (either through the CLI or the application).

Reply:
Oh I see, regardless of ID it gets a response, makes sense.